tests/project/urls.py

```diff
@@ -4,5 +4,6 @@
 urlpatterns = [
     path("digid/", include("digid_eherkenning.digid_urls")),
     path("eherkenning/", include("digid_eherkenning.eherkenning_urls")),
+    path("", include("digid_eherkenning.metadata_urls")),
```

Comment:
swap with the `admin` prefix to prevent accidental matches (unlikely, but cleaner)